blockly/src/level/BlocklyLevel.java

```diff
@@ -74,7 +75,7 @@ public Set<String> blockedBlocklyElements() {
    *
    * @return The Design of this level.
    */
-  public DesignLabel designLabel() {
-    return designLabel;
+  public Optional<DesignLabel> designLabel() {
+    return Optional.ofNullable(designLabel);
```

Comment:
habe das auch eben oben gesehe, vlt macht das mal sin später in DungeonLevel zu verschieben... Aber auch wieder hier die frage kann das null sein?

Reply:
tatsächlich ja.
Aktuell haben level bei uns kein Design, nur die Tiles.
Wenn beim Level aus 0-Tiles besteht, dann hat mein Level kein Design.